connectors-home/modules/ROOT/partials/configure-acb-connectors-partial.adoc

```diff
@@ -12,23 +12,22 @@ include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"]
 // tag::acb-connector-config-intro[]
 include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"]
 
-To use Anypoint Code Builder to configure a connector, create a basic integration project in Anypoint Code Builder, add connectors and other components to your Mule application to process your business logic, and configure attributes. 
+To use Anypoint Code Builder to configure a connector or module, create a basic integration project in Anypoint Code Builder, add connectors and other components to your Mule application to process your business logic, and configure the attributes. 
 
 == Before You Begin
 
 Before creating an integration project, you must:
 
 * xref:anypoint-code-builder::start-acb.adoc[Set up and access the Anypoint Code Builder web or desktop IDE]
 * Have credentials to access the connector's API
+* Ensure that the Visual Studio Code Extension Pack for Java is installed.
```

Comment:
The full name of the extension is "Extension Pack for Java" (from MicroSoft). It's an extension for Visual Studio Code. Otherwise, could sound like it's an extension from Visual Studio Code vs. for use in VS Code. https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack

Reply:
```suggestion
* Ensure that Extension Pack for Java is installed in Visual Studio Code.
```